dotnet/src/webdriver/ICookieJar.cs

```diff
@@ -43,19 +47,19 @@ public interface ICookieJar
         /// <param name="name">The name of the cookie to retrieve.</param>
         /// <returns>The <see cref="Cookie"/> containing the name. Returns <see langword="null"/>
         /// if no cookie with the specified name is found.</returns>
-        Cookie GetCookieNamed(string name);
+        Cookie? GetCookieNamed(string? name);
```

Comment:
We allow param to be `null`? I guess each cookie must have name.

Reply:
We can make it non-nullable as a hint to users, even if the method works with `null` (it will just find nothing)

Reply:
Actually, the implementation class `CookieJar` explicitly handles the `null` case. Maybe it's an allowed input?

![image](https://github.com/user-attachments/assets/31de02c1-3f89-432b-9a5a-253de474db61)

Reply:
All right :( Even in my opinion the API is incorrect, we shouldn't break it. So, allowing null as a cookie name :(

Reply:
Interesting, it even doesn't use spec remote end: https://www.w3.org/TR/webdriver2/#get-named-cookie

Reply:
The latest changes don't allow null, it's OK.